src/main/java/http/request/RequestRouter.java

```diff
@@ -7,33 +7,41 @@
 import java.nio.charset.StandardCharsets;
 
 
-import static http.request.RequestParser.*;
-
 public class RequestRouter {
     public static void requestReader(InputStream input) throws IOException {
```

Comment:
hmm, I'm not sure about the naming and single responsibility principle here - the `requestReader` sounds like a function that returns an instance of the `requestReader` instead of doing an action, while the action here is reading annd parsing, I wonder if this could be split into two functions (one for reading and one for calling parse)? What do you think?